math-scala/src/test/scala/org/apache/mahout/math/algorithms/ClusteringSuiteBase.scala

```diff
@@ -45,4 +48,17 @@ trait ClusteringSuiteBase extends DistributedMahoutSuite with Matchers {
     val epsilon = 1E-6
     (myAnswer.norm - correctAnswer.norm) should be <= epsilon
   }
+
+  test("inCore DBSCAN test") {
+//    import org.apache.mahout.math.algorithms.clustering._
+//
+//    val dm = DistanceMetricSelector.namedMetricLookup('Euclidean)
+////    val url: URL = getClass().getResource("./data21.txt")
+//    //TO DO //Need help
```

Comment:
@AdityaAS if you need some toy data files to for tests, you can place them in `$MAHOUT_HOME/examples/bin/resources".

Reply:
Ah. Thanks. I'll do that.

Reply:
I apologize @AdityaAS I'd not noticed that this `.txt` file wasthis short-  No need to move to `$MAHOUT_HOME/examples/bin/resources` for Unit test input.  FYI though if you write an example for this to be placed in `examples`, that's where you would place `.txt` files or any other resources.